conans/util/progress_bar.py

```diff
@@ -0,0 +1,69 @@
+
+import os
+from tqdm import tqdm
+from contextlib import contextmanager
+
+TIMEOUT_BEAT_SECONDS = 30
```

Comment:
Do we want to make this value configurable? It is used to print a heartbeat character instead of the progress bar. Which value to use?

Reply:
If that is for the "dot" being printed every few seconds, both for the user and CI not aborting, I would say hardcode it by now, to something that makes sense from a UI perspective (every 10 seconds?)

Reply:
This is the "dot" being printed only if `not output.is_terminal`, so tipically it will be shown in CI and tests.